### PR TITLE
Add LOAD_STATE_SLOT N command to stdin/network protocol

### PR DESCRIPTION
**Pull request by JoeOsborn**
This adds a LOAD_STATE_SLOT <slot number> command to the network and stdin protocols.  This way an external process can trigger loads of specific states.

### CODE REVIEW

File: command.c [hunk 1]
```diff
@@ -68,6 +68,8 @@
 
 #define CMD_BUF_SIZE           4096
 
+void command_post_state_loaded();
```
Comment (LibretroAdmin) [inline command.c:71]: Add 'void' in between parentheses.

File: command.c [hunk 2]
```diff
@@ -643,6 +645,45 @@ bool command_show_osd_msg(command_t *cmd, const char* arg)
...
+            sizeof(state_path));
+
+      core_serialize_size(&info);
+      savestates_enabled = (info.size > 0);
+   }
+   if (savestates_enabled)
+   {
+      ret = content_load_state(state_path, false, false);
+      if(ret)
+      {
```
Comment (LibretroAdmin) [inline command.c:673]: Get rid of one line brackets like this

File: command.c [hunk 3]
```diff
@@ -643,6 +645,45 @@ bool command_show_osd_msg(command_t *cmd, const char* arg)
...
+          command_post_state_loaded();
+      }
+   }
+   else
+   {
```
Comment (LibretroAdmin) [inline command.c:678]: Same here, get rid of one line code brackets.

**Review by LibretroAdmin — CHANGES_REQUESTED** [review]

File: command.c (hunk 3 above)
Comment (JoeOsborn) [reply command.c]: Sure thing.

File: command.c (hunk 2 above)
Comment (JoeOsborn) [reply command.c]: Got it!

File: command.c (hunk 1 above)
Comment (JoeOsborn) [reply command.c]: Done.

**Pull request merged**